rotkehlchen/tests/api/test_ethereum_transactions.py

```diff
@@ -1261,6 +1262,7 @@ def query_transactions(rotki):
         assert dbevents.get_history_events(cursor, HistoryEventFilterQuery.make(), True) == []
 
 
+@pytest.mark.skip()  # Unskip after Leftris' changes to the evm trasnactions endpoint
```

Comment:
Please in all those tests add a comment fine but don't make typos like "Leftris'"

I need all those greppable. And the field to use is this:
```suggestion
@pytest.mark.skip('LEFTERIS TODO')  # Unskip after Lefteris' changes to the evm transactions endpoint
```

Add this `LEFTERIS TODO` in all